src/Core/AbstractTest.php

```diff
@@ -182,6 +182,30 @@ public function setPending($state)
     }
 
     /**
+     * Set the focused status of the test and its children according to the
+     * supplied focus pattern and/or skip pattern
+     *
+     * @param string|null $focusPattern
+     * @param string|null $skipPattern
+     */
+    public function applyFocusPatterns($focusPattern, $skipPattern = null)
+    {
+        $title = $this->getTitle();
+
+        if ($skipPattern !== null && preg_match($skipPattern, $title)) {
```

Comment:
@brianium I don't think there's any more code to write, it's only really the docs. I've already document `fit()`, `fcontext()`, and `fdescribe()`, but not these new CLI options. Shouldn't take too long though.

I opened up another PR on the docs with the current progress: peridot-php/peridot-php.github.io#3